sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -65,6 +75,52 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
       FakeStudentUsersTypes.FakeStudentUserType1,
       dataSource,
     );
+    student.sinValidation = createFakeSINValidation(
+      {
+        student,
+      },
+      { initialValue: { isValidSIN: true } },
+    );
+    await db.student.save(student);
+  });
+
+  beforeEach(async () => {
+    const studentRestrictions = await db.studentRestriction.find({
+      select: { id: true, student: { id: true } },
+      relations: { student: true },
+      where: { student: { id: student.id } },
+    });
+    await db.studentRestriction.remove(studentRestrictions);
```

Comment:
No need to cleanup if we use different students following the recommendation from https://github.com/bcgov/SIMS/pull/3298/files#r1621517863